### ISSUE TITLE
mark stack overflow

### ISSUE THREAD

**Issue by kayceesrk** [ocamllabs/ocaml-multicore#25]
Run [queue_test](https://github.com/ocamllabs/reagents/blob/73e823908dfb6750ae9a227140bb02e682ad2580/test/queue_test.ml) example from reagents library as `queue_test.byte 4 1000000`. This produces the following output:

```
[0] items_per_domain = 250000
consumed=189249
consumed=285174
consumed=197374
consumed=328203
mark stack overflow
```

and terminates.

**Comment by kayceesrk** [comment]
PR for fixing this https://github.com/ocamllabs/ocaml-multicore/pull/68.

**Comment by stedolan** [comment]
This was fixed by #68
